### PR TITLE
Fatma practice

### PR DESCRIPTION
**Pull request by FatmaCG**
Hi my first commit

### CODE REVIEW

**Comment by FatmaCG** [comment]
Again reviewers

File: src/test/java/Practice_Fatma.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+public class Practice_Fatma {
+    public static void main(String[] args) {
+        System.out.println("Hello World");
+
+        System.out.println("My first GitHub Practice");
```
Comment (ademcigdem) [inline src/test/java/Practice_Fatma.java:5]: Please do clear code :)

File: src/test/java/Practice_Fatma.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+public class Practice_Fatma {
```
Comment (ademcigdem) [inline src/test/java/Practice_Fatma.java:1]: use proper class name

**Review by ademcigdem — APPROVED** [review]

File: src/test/java/Practice_Fatma.java (hunk 2 above)
Comment (FatmaCG) [reply src/test/java/Practice_Fatma.java:1]: ok

**Pull request merged**